akka-remote/src/main/scala/akka/remote/artery/ArterySettings.scala

```diff
@@ -4,22 +4,17 @@
 
 package akka.remote.artery
 
-import java.net.InetAddress
-
-import scala.concurrent.duration._
-
-import scala.annotation.nowarn
-import com.typesafe.config.Config
-import com.typesafe.config.ConfigFactory
-
 import akka.NotUsed
 import akka.japi.Util.immutableSeq
 import akka.stream.ActorMaterializerSettings
-import akka.util.Helpers.ConfigOps
-import akka.util.Helpers.Requiring
-import akka.util.Helpers.toRootLowerCase
+import akka.util.Helpers.{ toRootLowerCase, ConfigOps, Requiring }
 import akka.util.WildcardIndex
 import akka.util.ccompat.JavaConverters._
+import com.typesafe.config.{ Config, ConfigFactory }
+
+import java.net.{ Inet4Address, Inet6Address, InetAddress }
+import scala.annotation.nowarn
+import scala.concurrent.duration._
```

Comment:
We do non-combined imports on purpose because it makes the risk for merge conflicts a bit lower, so please revert the imports changes.